package/src/Slider.tsx

```diff
@@ -0,0 +1,273 @@
+     */
+    onSlidingComplete?: (value: number) => void;
+
+    /**
+     * Used to locate this view in UI automation tests.
+     */
+    testID?: string;
+
+    /**
+     * Sets an image for the thumb. Only static images are supported.
+     */
+    thumbImage?: ImageSource;
+
+    /**
+     * If true the slider will be inverted.
+     * Default value is false.
+     */
+    inverted?: boolean;
+
+    /**
+     * A string of one or more words to be announced by the screen reader.
+     * Otherwise, it will announce the value as a percentage.
+     * Requires passing a value to `accessibilityIncrements` to work correctly.
+     * Should be a plural word, as singular units will be handled.
+     */
+    accessibilityUnits?: string;
+
+    /**
+     * An array of values that represent the different increments displayed
+     * by the slider. All the values passed into this prop must be strings.
+     * Requires passing a value to `accessibilityUnits` to work correctly.
+     * The number of elements must be the same as `maximumValue`.
+     */
+    accessibilityIncrements?: Array<string>;
+  }>;
+
+const SliderComponent = (
+  props: Props,
+  forwardedRef?: Ref<typeof RCTSliderNativeComponent>,
+) => {
+  const style = StyleSheet.compose(styles.slider, props.style);
+
+  const {
+    onValueChange,
+    onSlidingStart,
+    onSlidingComplete,
+    onAccessibilityAction,
+    ...localProps
+  } = props;
+
+  const onValueChangeEvent = onValueChange
+    ? (event: Event) => {
+        onValueChange(event.nativeEvent.value);
+      }
+    : null;
+
+  const _disabled =
+    typeof props.disabled === 'boolean'
+      ? props.disabled
+      : props.accessibilityState?.disabled === true;
+
+  const _accessibilityState =
+    typeof props.disabled === 'boolean'
+      ? {...props.accessibilityState, disabled: props.disabled}
+      : props.accessibilityState;
+
+  const onSlidingStartEvent = onSlidingStart
+    ? (event: Event) => {
+        onSlidingStart(event.nativeEvent.value);
+      }
+    : null;
+  const onSlidingCompleteEvent = onSlidingComplete
+    ? (event: Event) => {
+        onSlidingComplete(event.nativeEvent.value);
+      }
+    : null;
+  const onAccessibilityActionEvent = onAccessibilityAction
+    ? (event: AccessibilityActionEvent) => {
+        onAccessibilityAction(event);
+      }
+    : null;
+
+  return (
+    <RCTSliderNativeComponent
+      {...localProps}
+      accessibilityState={_accessibilityState}
+      thumbImage={
+        Platform.OS === 'web'
+          ? props.thumbImage
+          : props.thumbImage
+          ? Image.resolveAssetSource(props.thumbImage)
+          : undefined
+      }
+      ref={forwardedRef}
+      style={style}
+      onChange={onValueChangeEvent}
+      onRNCSliderSlidingStart={onSlidingStartEvent}
+      onRNCSliderSlidingComplete={onSlidingCompleteEvent}
+      onRNCSliderValueChange={onValueChangeEvent}
+      enabled={!_disabled}
+      disabled={_disabled}
```

Comment:
This needs to be addressed, but in a separate PR.

Reply:
Yes, already discussed while migrating to new architecture - let's create an issue for that: #426